pkgs/development/python-modules/simplicial/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  wheel,
+  matplotlib,
+  numpy,
+  pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "simplicial";
+  version = "1.0.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "simoninireland";
+    repo = "simplicial";
+    rev = "v${version}";
+    hash = "sha256-o9YoswTthqW+xMrIrSMscRPnqz9A0EGQMXlbuIZ0OWE=";
+  };
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
```

Comment:
(nit)
The check phase happens after the build, so it might be better to move it after `pythonImportsCheck`